pkg/server/routes/tracking.g.go

```diff
@@ -15,7 +16,8 @@ func RegisterTrackingServiceRoutes(service service.TrackingService, parser *pars
 		if err := parser.ParseQuery(ctx, input); err != nil {
 			return err
 		}
-		output, err := service.GetExperimentByName(ctx.Context(), input)
+		logger := utils.GetLoggerFromContext(ctx.UserContext())
+		output, err := service.GetExperimentByName(utils.NewContextWithLogger(ctx.Context(), logger), input)
```

Comment:
Nitpick but in order to generate as little as possible code, could we shuffle the logger creation into a helper.

Something along the lines of:
```
output, err := service.GetExperimentByName(utils.NewContextWithLogger2(ctx), input)

// In utils.go

func NewContextWithLogger2(ctx) ctx { 
    logger := GetLoggerFromContext(ctx.UserContext())
    NewContextWithLogger(ctx.Context(), logger)
}
```

Unless, we are foreseeing that we will use `logger` inside some of the other generated code.

Reply:
It can wait until after your holidays if those have started already ;)